dataprofiler/tests/profilers/test_unstructured_text_profile.py

```diff
@@ -166,6 +166,18 @@ def test_case_sensitivity(self):
                                    'friends': 1}
             self.assertDictEqual(expected_word_count, profile['word_count'])
 
+        # change the merge order
+        with self.assertWarnsRegex(UserWarning,
+                "The merged Text Profile will not be case sensitive since there"
+                " were conflicting values for case sensitivity between the two "
+                "profiles being merged."):
+            text_profile3 = text_profile2 + text_profile1
+            profile = text_profile3.profile
+            # Assert word counts are correct
+            expected_word_count = {'hello': 1, 'name': 1, 'grant': 2, 'bob': 1,
+                                   'friends': 1}
+            self.assertDictEqual(expected_word_count, profile['word_count'])
```

Comment:
going to approve, but instead of rewriting the expected value, it would have been better to use the previous expectation to show that the result is not order dependent.

Additionally:
```python
profile = text_profile3.profile
# Assert word counts are correct
expected_word_count = {'hello': 1, 'name': 1, 'grant': 2, 'bob': 1,
                       'friends': 1}
self.assertDictEqual(expected_word_count, profile['word_count'])
```
did not need to be indented (I see it was previously). However, in future if we update, can indent back over.

Reply:
Also I don't know if this fix belongs in this PR, but I think this warning gets raised even if both profiles are not case sensitive, which feels incorrect because those values aren't "conflicting". The warning is around line 190 in `__add__`